rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+
+    # Add the rental to book's rentals
+    book.add_rental(self)
+    # Add the rental to person's rentals
+    person.add_rental(self)
```

Comment:
- [ ] Your `add_rental` method calls here do not match the way you defined them in the Book and Person classes. We don't need these methods this way.

  ```
  book.rentals << self

  person.rentals << self
  ```